Library/ViewModels/CategorySelectionViewModelTests.swift

```diff
@@ -377,6 +379,38 @@ final class CategorySelectionViewModelTests: TestCase {
     }
   }
 
+  func testPostNotification() {
+    let mockKVStore = MockKeyValueStore()
+
+    let categoriesResponse = RootCategoriesEnvelope.init(rootCategories: [
+      .art
+    ])
+
+    let artIndexPath = IndexPath(item: 0, section: 0)
+    let illustrationIndexPath = IndexPath(item: 1, section: 0)
+    let artId = Category.art.intID ?? 0
+    let illustrationId = Category.illustration.intID ?? 0
+
+    let mockService = MockService(fetchGraphCategoriesResponse: categoriesResponse)
+
+    withEnvironment(apiService: mockService, userDefaults: mockKVStore) {
+      self.vm.inputs.viewDidLoad()
+
+      self.scheduler.advance()
+
+      self.vm.inputs.categorySelected(with: (artIndexPath, artId))
+      self.vm.inputs.categorySelected(with: (illustrationIndexPath, illustrationId))
+
+      self.postNotification.assertDidNotEmitValue()
+      XCTAssertFalse(mockKVStore.hasCompletedCategoryPersonalizationFlow)
+
+      self.vm.inputs.continueButtonTapped()
+
+      self.postNotification.assertValueCount(1)
```

Comment:
Can we test which notification was posted?

Reply:
Ah yeah thanks for catching!